HW3/task2.py

```diff
@@ -0,0 +1,20 @@
+#Напишите программу, которая найдёт произведение пар чисел списка. Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+#Пример:
+#- [2, 3, 4, 5, 6] => [12, 15, 16];
+#- [2, 3, 5, 6] => [12, 15]
+
+from random import randint
+
+list=[]
+list2=[]
+number = 7 #int(input('Введите размер списка '))
+for i in range(number):
+    list.append(randint(1, 9))
+
+print(list)
+
+for i in range(0,(round(len(list)/2))):
+    m = list[i] * list[len(list)-i-1]
```

Comment:
Точно не очень хорошая идея использовать функцию округления таким образом. Время от времени у вас будут аккумуляция не так, как вам нужно для правильной работы функции `range`. Да и вообще, зачем считать длину списка, если у вас есть переменная, которая уже содержит его длину? И последнее – `0` передавать в `range ` не обязательно.
```suggestion
for i in range((number + 1) // 2):
    m = list[i] * list[number - i - 1]
```
Кстати, `number ` в расчёте можно и не указывать. Все равно будет работать, что с ним, что без него
```suggestion
for i in range((number + 1) // 2):
    m = list[i] * list[-i - 1]
```